automated install/basic-install.sh

```diff
@@ -1896,6 +1898,8 @@ installPihole() {
             # make the Web directory if necessary
             install -d -m 0755 ${webroot}
         fi
+        chmod a+rx /var/www
+        chmod a+rx /var/www/html
```

Comment:
This is partially taken care of below, and should anyways be within the `INSTALL_WEB_SERVER` if block.